static/cypress/fixtures/tasks.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "id": "ecg-qrs-classification-physiodb",
+    "description": "Labeling the QRS complex of Physionet ECGs",
+    "labels": [
+      "Atrial fibrillation",
+      "Normal sinus rhythm",
+      "Ventricular tachycardia",
+      "Noise",
+      "Other"
+    ]
+  }
+]
```

Comment:
Yeah no not really, that is not good. I looked for another way and found one, which can be applied to the other tests too. But since it required refactoring in some other places too, I put it in [another pr](https://github.com/team-folivora/annoto/pull/132), to not pollute this one